controllers/controllers.go

```diff
@@ -63,8 +63,52 @@ func (ctrl *OAuthController) EndpointHandler(w http.ResponseWriter, r *http.Requ
 	}
 }
 
+func (ctrl *OAuthController) tokenError(w http.ResponseWriter, err error) error {
+	data, statusCode, header := ctrl.Service.OauthServer.GetErrorData(err)
+	logrus.Errorf("%s: %s", data["error_description"], data["error"])
+	sentry.CaptureException(err)
```

Comment:
remove sentry for 401
(also https://github.com/getAlby/oauth2server/pull/61/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L40)